### PR TITLE
Article updates for new required fields

### PR DESCRIPTION
**Pull request by stephenwf**
@thewilkybarkid @elrossco22

### CODE REVIEW

File: data/articles/04395.json [hunk 1]
```diff
@@ -9,7 +9,9 @@
             "type": "research-article",
             "doi": "10.7554/eLife.04395",
             "title": "Mapping the zoonotic niche of Ebola virus disease in Africa",
+            "titlePrefix": "Title prefix",
```
Comment (thewilkybarkid) [inline data/articles/04395.json]: Could you remove this for all of them except 10627? It's only going to be used by certain article types.

File: data/articles/05770.json [hunk 2]
```diff
@@ -1,4 +1,6 @@
 {
+    "received": "2014-08-18",
+    "accepted": "2014-08-31",
```
Comment (stephenwf) [inline data/articles/05770.json]: Looks here like its been made required for the archive:
https://github.com/elifesciences/api-raml/commit/e5ba4ea1528f3f990494159df19e272254392e65#diff-28d9beff6cba1dc295568da1bd47802bR38

File: data/articles/04395.json (hunk 1 above)
Comment (stephenwf) [inline data/articles/04395.json]: Done!

File: data/articles/05770.json (hunk 2 above)
Comment (thewilkybarkid) [inline data/articles/05770.json]: Hmm, that's incorrect. I'll fix it.

Comment (stephenwf) [inline data/articles/05770.json]: Yeh no problem

**Comment by stephenwf** [comment]
@thewilkybarkid This should be ready to merge now

**Pull request merged**